src/main/java/frc/robot/commands/deployer/LowerDeployer.java

```diff
@@ -33,6 +35,7 @@ public boolean isFinished() {
     @Override
     public void end(boolean interrupted) {
         deployer.setDeployerMotorSpeed(0);
+        timeoutCounter.increaseTimeoutCount();
```

Comment:
This won't work as-is, you will need to check "isFinished" instead of incrementing every time in "end()"

Reply:
I have made these changes

Reply:
I have made these changes.